upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -70,11 +71,26 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 		Manifest: fi.String("dns-controller/v1.4.1.yaml"),
 	})
 
+	// TODO: Create configuration object for cni providers (maybe create it but orphan it)?
 	if b.cluster.Spec.Networking.Kopeio != nil {
 		key := "networking.kope.io"
 		version := "1.0.20161116"
 
-		// TODO: Create configuration object (maybe create it but orphan it)?
```

Comment:
Can you leave me my comment?  I'd rather copy and paste the comment, so that it doesn't get lost ... :-)  Feels more actionable...